plugins/Installation/Controller.php

```diff
@@ -112,15 +111,11 @@ function systemCheck()
             __FUNCTION__
         );
 
-        $view->duringInstall = true;
+        /** @var DiagnosticService $diagnosticService */
+        $diagnosticService = StaticContainer::get('Piwik\Plugins\Diagnostics\DiagnosticService');
+        $view->diagnosticReport = $diagnosticService->runDiagnostics();
```

Comment:
Stability means less bug reports, so I'd say it's good. A comment might be useful here to explain, though.